src/components/views/settings/tabs/user/SessionManagerTab.tsx

```diff
@@ -17,10 +17,17 @@ limitations under the License.
 import React from 'react';
 
 import { _t } from "../../../../../languageHandler";
+import SettingsSubsection from '../../shared/SettingsSubsection';
 import SettingsTab from '../SettingsTab';
 
 const SessionManagerTab: React.FC = () => {
-    return <SettingsTab heading={_t('Sessions')} />;
+    return <SettingsTab heading={_t('Sessions')}>
+        <SettingsSubsection
+            heading={_t('Current session')}
+            // TODO session content coming here
+            // in next PR
```

Comment:
```suggestion
            // TODO(kerrya): session content coming here
            // in next PR
```

or some sort of way to know who to track down if for some reason the TODO sits there for decades :)

alternative would be an issue link: "TODO: Implement as part of <link>"

Reply:
This one is already done in my next open PR https://github.com/matrix-org/matrix-react-sdk/pull/9151, but will do for future